tests/test_gaap.py

```diff
@@ -122,8 +122,8 @@ def makeAlgorithm(self, gaapConfig=None):
         if gaapConfig is None:
             gaapConfig = lsst.meas.extensions.gaap.SingleFrameGaapFluxConfig()
         gaapPlugin = lsst.meas.extensions.gaap.SingleFrameGaapFluxPlugin(gaapConfig,
-                                                                         'ext_gaap_GaapFlux',
-                                                                         schema, None)
+                                                                         "ext_gaap_GaapFlux",
+                                                                         schema, None, logName="testGaap.log")
```

Comment:
Might be a long-shot...but would it be possible to force a failure to `_solve()` in the test code to make sure it "does the right thing"?

Reply:
So, the `testFail` method does that, especially at [L266](https://github.com/lsst/meas_extensions_gaap/blob/cb7397a053f9f69d9587af2d6733b028f990b9e9/tests/test_gaap.py#L266) where `flag_gaussianization` is set to True by the `fail` method after logging. But running pytest or scons doesn't create the log file. Running the test script as `python tests/test_gaap.py` (not recommended and is going to be phased out) prints it on the command line. It looks as follows:

```
.ERROR: Unable to calculate psf matching kernel
measurement.ext_gaap_GaapFlux ERROR: Failed to solve for PSF matching kernel in GAaP for (100.000000, 670.000000): Problematic scaling factors = 100.0 Errors: Exception('Unable to determine kernel sum; 0 candidates')
ERROR: Unable to calculate psf matching kernel
measurement.ext_gaap_GaapFlux ERROR: Failed to solve for PSF matching kernel in GAaP for (100.000000, 870.000000): Problematic scaling factors = 100.0 Errors: Exception('Unable to determine kernel sum; 0 candidates')
ERROR: Unable to calculate psf matching kernel
measurement.ext_gaap_GaapFlux ERROR: Failed to solve for PSF matching kernel in GAaP for (-10.000000, -20.000000): Problematic scaling factors = 100.0 Errors: Exception('Unable to determine kernel sum; 0 candidates')
```

Any inputs on how to run the test_gaap.py in a manner to produce a log file? I was going to run `forcedPhotCoadd` with just the logging changes on the `ci_imsim` and `ci_hsc` branches and see how the error messages look like, but it is of course not principled.

Reply:
Hhhmmm...no, I have no idea how to do that.  It seems there are now no GAaP failures in the limited ci datasets (which is fundamentally a very good thing if the numbers are all sensible!)  But what you have shown here does sufficiently scratch the itch I had to see an example of the log output on error, so thanks!

Reply:
`.log` seems odd for the name of a logger -- it's not a file name.

Reply:
I've removed specifying any value for `logName`, since the Python logging can handle `None`. The logging aspects themselves are checked in the `testFail` method explicitly.